test/preprocessors/TypeScriptInterfacePreprocessor.test.ts

```diff
@@ -0,0 +1,28 @@
+import TypeScriptInterfacePreprocessor from "../../src/preprocessors/TypeScriptInterfacePreprocessor";
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+import FilePath from "../../src/model/dataTypes/FilePath";
+import EntityCollection from "../../src/model/dataTypes/EntityCollection";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+
+describe("TypeScriptInterfacePreprocessor", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)